lib/plugins/aws/invokeLocal/index.js

```diff
@@ -30,7 +30,7 @@ const ensureRuntimeWrappers = isStandalone
         await fse.copy(path.resolve(__dirname, 'runtimeWrappers'), runtimeCachePath);
         return fse.ensureFile(path.resolve(runtimeCachePath, 'validationFile'));
       })
-  : () => Promise.resolve(__dirname);
+  : () => __dirname;
 
 class AwsInvokeLocal {
```

Comment:
In this class there are still functions that return promises but aren't marked as async, for example: `invokeLocal` and corresponding hook. Could you please adjust that as well?